validation/validation_test.go

```diff
@@ -17,54 +22,92 @@ var (
 	runtime = "runc"
 )
 
+type validation struct {
+	test        func(string, string, *rspec.Spec) error
+	description string
+}
+
 func init() {
 	runtime = os.Getenv("RUNTIME")
 }
 
-func runtimeValidate(runtime string, g *generate.Generator) error {
+func TestValidateRuntimeInside(t *testing.T) {
+	g, err := getDefaultGenerator()
+	if err != nil {
+		t.Errorf("%s failed validation: %v", runtime, err)
+	}
+	g.SetProcessArgs([]string{"/runtimetest"})
+
+	if err := runtimeInsideValidate(runtime, g); err != nil {
+		t.Errorf("%s failed validation: %v", runtime, err)
+	}
+}
+
+func TestValidateRuntimeOutside(t *testing.T) {
+	g, err := getDefaultGenerator()
+	if err != nil {
+		t.Errorf("%s failed validation: %v", runtime, err)
+	}
+
+	if err := runtimeOutsideValidate(runtime, g); err != nil {
+		t.Errorf("%s failed validation: %v", runtime, err)
+	}
+}
+
+func runtimeInsideValidate(runtime string, g *generate.Generator) error {
 	// Find the runtime binary in the PATH
 	runtimePath, err := exec.LookPath(runtime)
 	if err != nil {
 		return err
 	}
 
-	// Setup a temporary test directory
-	tmpDir, err := ioutil.TempDir("", "ocitest")
+	bundleDir, rootfsDir, err := prepareBundle(g)
 	if err != nil {
 		return err
 	}
-	defer os.RemoveAll(tmpDir)
+	//	defer os.RemoveAll(bundleDir)
```

Comment:
I'm guessing this change was for local testing, and that we want to restore the deferred removal before this PR lands?